app/static/scripts/countdown.js

```diff
@@ -39,5 +39,5 @@ function initClock(id, endtime) {
   var timeInterval = setInterval(updateClock, 1000);
 }
 
-var deadline = 'Sun Nov 04 2018 23:59:59 GMT-0400 (Eastern Daylight Time)';
+var deadline = 'Sun Mar 11 2019 23:59:59 GMT-0400 (Eastern Daylight Time)';
```

Comment:
There's some changes that really should be made about the game creation/game start process, and this would be affected by it, but also we should actually have this read from the game start date property instead of setting it manually. For now, I'd like to ask you if you could keep these changes out of the repo for now while I file an issue about this, and just have it handled on prod in the meantime.